FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentMiragePrismPrismBox.cs

```diff
@@ -21,28 +21,36 @@ public unsafe partial struct AgentMiragePrismPrismBox {
 public unsafe partial struct MiragePrismPrismBoxData {
-    [FieldOffset(0x08), FixedSizeArray] internal FixedSizeArray800<PrismBoxItem> _prismBoxItems;
-    [FieldOffset(0x1A908)] public PrismBoxItem TempContextItem;
-    [FieldOffset(0x1A990), FixedSizeArray] internal FixedSizeArray50<int> _pageItemIndexes;
-    [FieldOffset(0x1AA58)] public int TempContextItemIndex;
-    [FieldOffset(0x1AA5C)] public int SelectedPageIndex;
-    [FieldOffset(0x1AA60)] public int UsedSlots;
+    [FieldOffset(0x08), FixedSizeArray] internal FixedSizeArray8000<PrismBoxItem> _prismBoxItems;
+    [FieldOffset(0x109A08)] public PrismBoxItem TempContextItem;
 
-    [FieldOffset(0x1AA70)] public int CrystallizeCategory;
-    [FieldOffset(0x1AA74)] public int CrystallizeItemIndex;
-    [FieldOffset(0x1AA78)] public int CrystallizeItemCount;
-    [FieldOffset(0x1AA7C), FixedSizeArray] internal FixedSizeArray140<PrismBoxCrystallizeItem> _crystallizeItems;
-    [FieldOffset(0x1B9CC)] public PrismBoxCrystallizeItem CrystallizeSelectedItem;
+    // 7.1: 6,400 (0x1900) bytes were added here
 
-    [FieldOffset(0x1B9F4)] public uint FilterFlags;
-    [FieldOffset(0x1B9F8)] public AgentCabinet* AgentCabinet;
-    [FieldOffset(0x1BA00)] public AgentMiragePrismMiragePlate* AgentMiragePrismMiragePlate;
-    [FieldOffset(0x1BA08)] public byte FilterLevel;
-    [FieldOffset(0x1BA0A)] public byte FilterSex;
-    [FieldOffset(0x1BA10)] public Utf8String FilterString;
-    [FieldOffset(0x1BA78)] public Utf8String SearchString;
+    [FieldOffset(0x10B390), FixedSizeArray] internal FixedSizeArray50<int> _pageItemIndexes;
+    [FieldOffset(0x10B458)] public int TempContextItemIndex;
+    [FieldOffset(0x10B45C)] public int SelectedPageIndex;
+    [FieldOffset(0x10B460)] public int UsedSlots;
+
+    // 7.1: a new 32-bit int was added between UsedSlots and CrystallizeCategory
+
+    [FieldOffset(0x10B478)] public int CrystallizeCategory;
+    [FieldOffset(0x10B47C)] public ushort CrystallizeItemIndex;
+    [FieldOffset(0x10B47E)] public ushort CrystallizeItemCount;
+    [FieldOffset(0x10B484), FixedSizeArray] internal FixedSizeArray140<PrismBoxCrystallizeItem> _crystallizeItems;
+    [FieldOffset(0x10C3D4)] public PrismBoxCrystallizeItem CrystallizeSelectedItem;
+    [FieldOffset(0x113E9C)] public byte CrystallizeFilterFlags;
+
+    [FieldOffset(0x113E9D)] public byte SortType; // 0 = Descending, 2 = Ascending
+    [FieldOffset(0x113EB0)] public byte FilterLevel; // 0 = Unspecified = Max Level
+    [FieldOffset(0x113EB2)] public byte FilterSex; // 0 = Unspecified, 1 = Current, 2 = Male, 3 = Female
+    [FieldOffset(0x113EB8)] public Utf8String FilterString; // Inline buffer only
+    [FieldOffset(0x113F20)] public Utf8String SearchString; // Inline buffer only
+
+    // The following pointers exist but shouldn't be used over AgentX.Instance():
+    // OnShow: AgentCabinet* is stored at +0x113EA0
+    // OnShow: AgentMiragePrismMiragePlate* is stored at +0x113EA8
```

Comment:
You could add the `AgentCabinet` and `AgentMiragePrismMiragePlate` pointers as `internal`, so that they can be exported with CExporter.

```suggestion
    [FieldOffset(0x113EA0)] internal AgentCabinet* AgentCabinet;
    [FieldOffset(0x113EA8)] internal AgentMiragePrismMiragePlate* AgentMiragePrismMiragePlate;
```